cdk/.eslintrc.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+  env: {
+    browser: true,
```

Comment:
Should this be `false`? As in, are we ever expecting to be running in the browser?

Reply:
Good point, yes it should be! Will change it then merge, thanks.